src/api/db/structure.sql

```diff
@@ -698,20 +698,19 @@ CREATE TABLE `package_kinds` (
 CREATE TABLE `packages` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `project_id` int(11) NOT NULL,
-  `name` varchar(200) COLLATE utf8_bin NOT NULL,
+  `name` text COLLATE utf8_bin,
```

Comment:
Good spot Henne. This is an example of problem 2 in issue #2840. I think we all are guilty of making this mistake sometimes..